config/default/core.extension.yml

```diff
@@ -49,7 +49,6 @@ module:
   entity_reference_revisions: 0
   entity_usage: 0
   entitygroupfield: 0
-  eu_cookie_compliance_gtm: 0
```

Comment:
No longer in use? Fathom Analytics replaces it?

Reply:
I don't know why it has been removed, but not due to Fathom Analytics. It isn't on Gov

Reply:
I don't know why it has been removed, but not due to Fathom Analytics. It isn't on Gov

Reply:
Checked Andy's notes and it's not required.